packages/canvas/render/src/application-function/utils.ts

```
@@ -0,0 +1,74 @@
+  name: string
+  type: 'function' | string
+  content: {
+    exportName?: string
+    [props: string]: any
+  }
+  [props: string]: any
+}
+
+export function useUtils(context: Record<string, any>) {
+  const refreshKey = ref<number>(0)
+  const utils: Record<string, Function | any> = {}
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)